controls/bsi_app_4_4.yml

```diff
@@ -425,18 +425,45 @@ controls:
     levels:
     - elevated
     description: >-
-      Pods SHOULD ONLY be able to communicate with each other through the necessary network
-      ports, even within a Kubernetes namespace. There SHOULD be rules within the CNI that
-      disallow all but the necessary network connections within the Kubernetes namespace. These
+      (1) Pods SHOULD ONLY be able to communicate with each other through the necessary network
+      ports, even within a Kubernetes namespace. (2) There SHOULD be rules within the CNI that
+      disallow all but the necessary network connections within the Kubernetes namespace. (3) These
       rules SHOULD precisely define the source and destination of the allowed connections using at
       least one of the following criteria: service name, metadata (“labels”), Kubernetes service
       accounts, or certificate-based authentication.
-      All the criteria used as labels for a connection SHOULD be secured in such a way that they can
-      only be changed by authorised persons and management services.
+      (4) All the criteria used as labels for a connection SHOULD be secured in such a way that they
+      can only be changed by authorised persons and management services.
     notes: >-
-      TBD
-    status: pending
-    rules: []
+      In a cluster using a network plugin that supports Kubernetes network policy, network isolation
+      is controlled entirely by NetworkPolicy objects. In OpenShift, the default plugins (OpenShift SDN,

```

Comment:
we should remove OpenShift SDN as it is deprecated

Reply:
Removed it in the the latest commit